components/hint-message/hint-message.jsx

```diff
@@ -8,7 +8,8 @@ const HintMessage = (props) => {
         <h2><img src={props.imgSrc} /></h2>
         <h2>{props.header}</h2>
         {props.children}
-        <Link to={props.btn.to} className="btn">{props.btn.text}</Link>
+        { props.btn ? <Link to={props.btn.to} className="btn">{props.btn.text}</Link>
```

Comment:
kinda worried about the use of `btn` as variable name. Can we name this after what we use it for? Because this code suggests it should be called `internalLink`, controlling whether something is a react-router link or html link.